learn/guides/graphql-client-tool.md

```diff
@@ -76,7 +146,7 @@ extensions:
 
 The example below demonstrates how to generate multiple Ballerina clients from a GraphQL config file configured with a GraphQL schema (SDL) and multiple GraphQL documents.
```

Comment:
AFAIR, We decided not to include multi-client case. @hevayo Please confirm

Reply:
Had a discussion with @shafreenAnfar and included this section under `Advanced use cases`.